requirements.md

```diff
@@ -1,42 +1,50 @@
 # Introduction
 The purpose of this project is to test an applicant's coding and software design abilities.
 To that end, you may use online documentation and resources just as you would in your
- day to day work.  This project is designed to be completed within a few hours.
- 
+day-to-day work.  This project is designed to be completed within a few hours.
+
 # Project Story
 Emsi has large collections of documents and performs daily ingestion of new documents.
-The processing pipelines for these documents are designed to be streaming such that 
-only one or a few documents are memory at a time.  You have been asked to build a
+The processing pipelines for these documents are designed to be streaming such that
+only one or a few documents are in memory at a time.  You have been asked to build a
 component in such a pipeline for processing job postings scraped from the Internet.
 The purpose of this component is to tag each document with a Standard Occupation Code
- based on the already present ONET code.
- 
+based on the already present ONET code, compile some summary statistics on the
+dataset, and reformat the data as a SQLite database.
+
 The sample data (`sample.gz`) is gzipped JSON lines.  Each line is a JSON object describing
-a job posting.  The input objects should be mapped to lines of JSON objects with the following schema:
+a job posting.  The input objects should be mapped to rows in a SQLite table with the following schema:
 
 ```
 {
-	body: string,
-	title: string,
-	expired: string,
-	posted: string,
-	state: string,
-	city: string,
-	onet: string,
-	soc5: string,
+    body: TEXT,
+    title: TEXT,
+    expired: DATE,
+    posted: DATE,
+    state: TEXT,
+    city: TEXT,
+    onet: TEXT,
+    soc5: TEXT,
+    soc2: TEXT
 }
 ```
 
-Most fields should simply be copied across, with two exceptions:
+Most fields should simply be copied across, with three exceptions:
 * `body` should be the input `body` stripped of everything that looks like an HTML tag.
-* `soc5` should be injected based on the input `onet` and the associated mapping described 
-         in `map_onet_soc.csv`.  Empty or unmapped `onet` values should map to `null`.
+* `soc5` should be injected based on the input `onet` and the associated mapping described in `map_onet_soc.csv`.
+* `soc2` should be injected based on soc5 and `soc_hierarchy.csv`.
```

Comment:
There should be a description here of what "soc2" and "soc5" actually are, and the fact that 5-digit codes are subsets of 2-digit codes.